plasmapy/utils/checks.py

```diff
@@ -272,6 +209,13 @@ def _check_quantity(arg, argname, funcname, units, can_be_negative=True,
 
     # Make sure arg is a quantity with correct units
 
+    unit_casting_warning = dedent(
+            f"""No units are specified for {argname} = {arg} in {funcname}. Assuming units of {str(units[0])}.
```

Comment:
Does this print nicely? I think there is a difference between triple quotes and having multiple single quote strings wrapped in parentheses in terms of how they handle newlines.

Reply:
I thought it printed okay. The snippet I pasted into the main pull request post was straight from the terminal, but I can provide a screenshot later today if you like.